packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -293,29 +293,32 @@ const Alert = ({
   }
 
   return (
-    <StyledAlert
-      intent={intent}
-      contrastType={contrastType}
-      isFullWidth={isFullWidth}
-      isDesktop={isDesktop}
-      textAlign={'left' as never}
+    <BaseBox
```

Comment:
I think all Box wrappers should have `display: inline-block` by default right? otherwise they'll take up full width even when the component inside it is inlined

Reply:
Better to keep it unset by default? Since we allow people to change the display prop on the wrapper, they can tweak it however they want. 

Maybe some specific components that needs to have a specific display prop, like `inline`, we can replicate that on the wrapper component by default but allow consumers to override it anyway. Let me see if I can find any such instances in this PR

Reply:
Did this for amount, other places it seems to not need it. Can you check once?

Reply:
Can we do it on Counter, Badge, Indicators too? components that are small in width and usually used along with text by default?

Reply:
So Counter, Badge & Indicators have flex wrappers so it shouldn't be much of a problem even if they make it inline

Reply:
Counter, Badge, & Indicators have been flex since forever now. With this change, they can now make it inline. I think its better to keep the way it is for now without changing it. I will make the wrapper also flex so that it remains as it was before

Reply:
Especially because these components are used within Card, Modal & Bottomsheet. Don't want to impact it there by enforcing it as inline here

Reply:
Oh actually we did try to make Badge inline-flex but I just noticed that we didn't change display property of the wrapper container https://github.com/razorpay/blade/pull/1254

We can do this separately then but we should make it inline-flex / inline-block as it was raised by our consumers as well https://github.com/razorpay/blade/issues/1241, https://razorpay.slack.com/archives/CMQ3RBHEU/p1685526734598399?thread_ts=1685526177.654729&cid=CMQ3RBHEU.

Lets take it up separately since this change is big in itself already.

Reply:
lets create an issue to track it?

Reply:
Saurabh already did #1447